### DISCUSSION TITLE
Add the option to restrict interaction with the color wheel unless the pointer event collides with the color

### DISCUSSION THREAD

**Discussion by mheirendt** [jaames/iro.js#181] [Feature Requests]
On an iPhone, you can drag up from the bottom of the phone to exit an application. When the color wheel is on the bottom of the screen, a color is selected just before exiting. It would be great to be able to restrict the selection of an active color unless the pointer event is within a configured  buffer of the color's position.

**Comment by jaames** [comment]
sorry, struggling to understand this one a little ':) 

you want to be able to click somewhere on the colorpicker without the last selected handle jumping to that position? or you want a timeout on input events?

**Comment by mheirendt** [comment]
No worries! You got it right with the first option, basically to only allow
drag and drop for the handles.

**Comment by jaames** [comment]
ah! understood, thanks for the clarification. I think I should be able to add this in a future update as an optional feature :)

**Comment by yogaaltariz** [comment]
has this been implemented?
